front_end_project/src/pages/resources/iotResources.js

```diff
@@ -1,102 +1,69 @@
-import { Container, Row, Col } from "react-bootstrap";
 import React, { Component } from "react";
```

Comment:
Just an FYI, you would not need to import React and components to every component since React v17

Refer: [Importing React](https://stackoverflow.com/questions/71725865/do-i-need-to-import-react-in-every-file)